lib/debt_management_center/workers/va_notify_email_job.rb

```diff
@@ -6,12 +6,12 @@ class VANotifyEmailJob
     include SentryLogging
     sidekiq_options retry: 14
 
-    def perform(email, template_id, personalisation = nil)
+    def perform(icn, template_id, personalisation = nil)
```

Comment:
This worker is also used by `financial_status_report_service` to send confirmation emails. Should either provide the ability to use both email and icn or update other references to use icn instead of email